src/main/java/com/github/manolo8/darkbot/gui/tree/editors/ImageOpener.java

```diff
@@ -0,0 +1,72 @@
+package com.github.manolo8.darkbot.gui.tree.editors;
+
+import com.github.manolo8.darkbot.config.ImageWrapper;
+import com.github.manolo8.darkbot.gui.AdvancedConfig;
+import eu.darkbot.api.config.util.OptionEditor;
+import com.github.manolo8.darkbot.gui.utils.Strings;
+import eu.darkbot.api.config.ConfigSetting;
+
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import java.awt.*;
+import java.io.File;
+
+public class ImageOpener extends JButton implements OptionEditor<ImageWrapper> {
+    private JFileChooser fc;
+    private ImageWrapper image;
+
+    public ImageOpener(){
+        putClientProperty("JButton.buttonType", "square");
+        addActionListener(e -> {
+            if (fc == null) {
+                fc = new JFileChooser(new File(".")) {
+                    @Override
+                    protected JDialog createDialog(Component parent) throws HeadlessException {
+                        JDialog dialog = super.createDialog(parent);
+                        dialog.setAlwaysOnTop(true);
+                        return dialog;
+                    }
+                };
+                FileNameExtensionFilter filter = new FileNameExtensionFilter(
+                        "PNG, JPG & GIF Images", "png","jpeg","jpg", "gif");
```

Comment:
```suggestion
                        "PNG, JPG & GIF Images", "png", "jpeg", "jpg", "gif");
```
are you sure that  "PNG, JPG & GIF Images" is a valid filter for this? (i have no clue but it looks off)

Reply:
theses are what i have tested with so far and seem to work, need to test with bmp but this covers the majority